package/MDAnalysis/analysis/rms.py

```diff
@@ -587,7 +587,7 @@ def _prepare(self):
             #    (x~: selected coordinates, x: all coordinates)
             # Final transformed traj coordinates: x' = (x-x~_com)*R + ref_com
             self._rot = np.zeros(9, dtype=np.float64)  # allocate space
-            self._R = np.matrix(self._rot.reshape(3, 3))
+            self._R = np.array(self._rot.reshape(3, 3))
```

Comment:
Should we note the change in the docstring?

Reply:
I don't think so, this change should mostly be invisible

Not sure you need to `np.array` call

Reply:
done